sample/shared/compose/src/commonMain/kotlin/com/arkivanov/sample/shared/dialog/DialogContent.kt

```diff
@@ -0,0 +1,31 @@
+package com.arkivanov.sample.shared.dialog
+
+import androidx.compose.material.Text
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+
+@Composable
+fun DialogContent(dialogComponent: DialogComponent) {
+    AlertDialog(
+        onDismissRequest = {
+            dialogComponent.onDismissClicked()
+        },
+        title = {
+            Text(text = "Decompose Sample Dialog")
+        },
+        text = {
+            Text(text = dialogComponent.message)
+        },
+        confirmButton = {},
+        dismissButton = {
```

Comment:
The Dismiss button is optional, whereas the Confirm button isn't. Maybe just use the Confirm button to dismiss the dialog?

Reply:
fixed